components/Drop/DropProgress.tsx

```diff
@@ -0,0 +1,77 @@
+import { Flex, Progress, SimpleGrid, Text } from '@chakra-ui/react'
+import BigNumber from 'bignumber.js'
+import useTranslation from 'next-translate/useTranslation'
+import numbro from 'numbro'
+import { FC, useMemo } from 'react'
+
+type Props = {
+  drops: {
+    minted: string
+    supply: string | null
+  }[]
+}
+
+const DropProgress: FC<Props> = ({ drops }) => {
+  const { t } = useTranslation('components')
+
+  const totalMinted = useMemo(
+    () =>
+      drops.reduce((acc, drop) => {
+        return acc + BigNumber(drop.minted).toNumber()
+      }, 0),
+    [drops],
+  )
+
+  const totalSupply = useMemo(
+    () =>
+      drops.reduce((acc, drop) => {
+        if (drop.supply === null) return Infinity
+        return acc + BigNumber(drop.supply).toNumber()
+      }, 0),
+    [drops],
+  )
+
+  const mintPercentage = useMemo(() => {
+    if (!totalSupply) return
+    return new BigNumber(totalMinted)
+      .div(BigNumber(totalSupply))
```

Comment:
Can pass string, number and BigNumber types on any functions on BigNumber 
```suggestion
      .div(totalSupply)
```